pkg/apis/performance/v1alpha1/performanceprofile_types.go

```diff
@@ -55,11 +56,57 @@ type RealTimeKernel struct {
 	Enabled *bool `json:"enabled,omitempty"`
 }
 
+// PerformanceProfileConditionType valid conditions of a PerformanceProfile.
+type PerformanceProfileConditionType string
+
+const (
```

Comment:
there are some standard condition types we can probably align with pretty easily.

from the conditions pkg, we have these, which are actually interpreted by the olm to mean something when performing updates and checking availability. 

```
const (         
        // ConditionAvailable indicates that the resources maintained by the operator,
        // is functional and available in the cluster.
        ConditionAvailable ConditionType = "Available"
                
        // ConditionProgressing indicates that the operator is actively making changes to the resources maintained by the
        // operator     
        ConditionProgressing ConditionType = "Progressing"
                
        // ConditionDegraded indicates that the resources maintained by the operator are not functioning completely.
        // An example of a degraded state would be if not all pods in a deployment were running.
        // It may still be available, but it is degraded
        ConditionDegraded ConditionType = "Degraded"
                        
        // ConditionUpgradeable indicates whether the resources maintained by the operator are in a state that is safe to upgrade.
        // When `False`, the resources maintained by the operator should not be upgraded and the
        // message field should contain a human readable description of what the administrator should do to
        // allow the operator to successfully update the resources maintained by the operator.
        ConditionUpgradeable ConditionType = "Upgradeable"
)
```

available=true
progressing=false
degraded=false
upgradable=true

^ that would indicate everything is stable.

Reply:
done

Reply:
I still see updated, updating and degraded, forgot to push? :)

Reply:
worked on it:) done